### PR TITLE
update readme and pediacluster example

### PR DESCRIPTION
**Pull request by Iceber**
update readme and *pediacluster* example https://github.com/clusterpedia-io/clusterpedia/pull/27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,10 +149,10 @@ metadata:
   name: cluster-example
 spec:
   apiserverURL: "https://10.30.43.43:6443"
-  caData: ""
-  tokenData: ""
-  certData: ""
-  keyData: ""
+  caData:
```
Comment (wzshiming) [inline README.md:152]: There is no need to remove this, `""` is still valid []byte

Comment (Iceber) [reply README.md:152]: but I got an error:
```
# pediaclusters.clusters.clusterpedia.io "cluster-x" was not valid:
# * spec.tokenData: Invalid value: "": spec.tokenData in body must be of type byte: ""
```

Comment (wzshiming) [reply README.md:152]: Well, I thought this validation rule was the same as JSON of Golang

Comment (wzshiming) [reply README.md:152]: LGTM

**Pull request merged**